src/signals/incident-management/containers/Dashboard/components/Filter/Filter.test.tsx

```diff
@@ -0,0 +1,247 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright (C) 2023 Gemeente Amsterdam
+
+import { act, fireEvent, render, screen } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import * as reactRedux from 'react-redux'
+import departmentsFixture from 'utils/__tests__/fixtures/departments.json'
+
+import { Filter } from './Filter'
+
+const mockCallback = jest.fn()
+
+window.HTMLElement.prototype.scrollIntoView = jest.fn()
+
+const departments = {
+  ...departmentsFixture,
+  count: departmentsFixture.count,
+  list: departmentsFixture.results,
+}
+
+describe('FilterComponent', () => {
+  beforeEach(() => {
+    jest.spyOn(reactRedux, 'useSelector').mockReturnValue(departments)
+  })
+
+  it('should render properly and select the first option', () => {
+    render(<Filter callback={mockCallback} />)
+
+    userEvent.click(screen.getAllByRole('listbox')[0])
+    expect(
+      screen.getByRole('listbox', { name: 'Actie Service Centr...' })
+    ).toBeInTheDocument()
+
+    expect(
+      screen.queryByRole('listbox', {
+        name: 'Amsterdamse Bos',
+      })
+    ).not.toBeInTheDocument()
+  })
+
+  it('should clear filterActiveName when clicking outside Filter', () => {
```

Comment:
Don't think this test tests what it should, since the option "Amsterdamse Bos" is not in the document inbetween the two clicks.